mscore/album.cpp

```diff
@@ -127,16 +128,61 @@ bool Album::createScore(const QString& fn)
       loadScores();
 
       Score* firstScore = _scores[0]->score;
-      if (!firstScore)
+      if (!firstScore) {
+            qDebug("First score is NULL. Will not attempt to join scores.");
             return false;
+            }
+
+      // do layout for first score's root and excerpt scores
       firstScore->doLayout();
+      for (int i=0; i<firstScore->excerpts().count(); i++) {
```

Comment:
Could you fix the spaces  here?
`
for (int i = 0; i < firstScore->excerpts().count(); i++) 
`